@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -93,12 +93,17 @@ const DEFAULT_MAX_NUM_SECOND_CHANCE_ELEMENTS = 300;
  * Maximum distance for text labels expressed as a ratio of distance to from the camera (0) to the
  * far plane (1.0). May be synchronized with fog value ?
  */
-const DEFAULT_MAX_DISTANCE_RATIO_FOR_TEXT_LABELS = 0.99;
+const DEFAULT_MAX_DISTANCE_RATIO_FOR_LABELS = 0.99;
 
 /**
- * Distance to the camera (range: `[0.0, 1.0]`) from which label start to scale.
+ * Minimum scaling factor that may be applied to labels when their are distant from focus point.
  */
-const DEFAULT_LABEL_SCALE_START_DISTANCE = 0.4;
+const DEFAULT_LABEL_DISTANCE_SCALE_MIN = 0.7;
+
+/**
+ * Maximum scaling factor that may be applied to labels due to their distance from focus point.
+ */
+const DEFAULT_LABEL_DISTANCE_SCALE_MAX = 1.5;
```

Comment:
Same as above.

Reply:
Please see above. I could not comment-link that property there, because it is "private" in scope of TextElementsRenderer.